portal/templates/instance_profile.html

```diff
@@ -102,6 +112,7 @@ <h1>Detailed services information currently not available</h1>
   </div>
 </section>
 
+<meta id="instance-data" data-name="{{name}}">
```

Comment:
this line together with line 128, 129 is used to get the name(id) of the instance